File: Template/Python/concrete_class_b.py
from abstract_class import AbstractClass

class ConcreteClassB(AbstractClass):
    def __init__(self) -> None:
        pass

    def _primitive_operation_1(self) -> None:
        print("primitive operation 1 b")

    def _primitive_operation_2(self) -> None:
        print("primitive operation 2 b")

File: Template/Java/src/com/yuriy/ConcreteClassB.java
package com.yuriy;

public class ConcreteClassB extends AbstractClass {
    @Override
    protected void primitiveOperation1() {
        System.out.println("Implementing operation 1 in B way");
    }

    @Override
    protected void primitiveOperation2() {
        System.out.println("Implementing operation 2 in B way");
    }
}
